### ISSUE TITLE
mdl v0.10.0 **with** the `-g` flag dumps a traceback

### ISSUE THREAD

**Issue by ben-alkov** [markdownlint/markdownlint#344]
## Description
mdl v0.10.0 **with** the `-g` flag dumps a traceback

### Environment
It's broken in [Travis](https://travis-ci.org/github/containerbuildsystem/osbs-client/jobs/716909367) (Centos 7)/[GH action](https://github.com/containerbuildsystem/atomic-reactor/runs/971406132?check_suite_focus=true) (Fed 32)/locally (Fed 32)

N.B. This *only* occurs when using `-g`

**MDL Version**
This definitely broke with v0.10.0 available via `gem update mdl`.

### Expected Behavior
`mdl -g .` finishes without error, and returns no hits (in the repo from the example below, at least)

### Actual Behavior
The context here is that I'm running locally in the root of the '[atomic-reactor](https://github.com/containerbuildsystem/atomic-reactor)' git repo.

```bash
10:02 🐚 mdl -g .
Traceback (most recent call last):
	7: from /home/USER/bin/mdl:23:in `<main>'
	6: from /home/USER/bin/mdl:23:in `load'
	5: from /home/USER/.gem/ruby/gems/mdl-0.10.0/bin/mdl:10:in `<top (required)>'
	4: from /home/USER/.gem/ruby/gems/mdl-0.10.0/lib/mdl.rb:80:in `run'
	3: from /home/USER/.gem/ruby/gems/mdl-0.10.0/lib/mdl.rb:80:in `each'
	2: from /home/USER/.gem/ruby/gems/mdl-0.10.0/lib/mdl.rb:82:in `block in run'
	1: from /home/USER/.gem/ruby/gems/mdl-0.10.0/lib/mdl/doc.rb:50:in `new_from_file'
/home/USER/.gem/ruby/gems/mdl-0.10.0/lib/mdl/doc.rb:50:in `read': No such file or directory @ rb_sysopen - CONTRIBUTING.md (Errno::ENOENT)
```
## Replication Case
`mdl -g .` on CentOS 7 or Fedora 32 (at least that's what I've tested with; there may be others)

**Comment by jaymzh** [comment]
This is a dupe of https://github.com/markdownlint/markdownlint/issues/339 - which is fixed in master, just haven't made a release yet.
